runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -773,6 +773,13 @@ TR_RelocationRuntime::validateAOTHeader(TR_FrontEnd *fe, J9VMThread *curThread)
    return false;
    }
 
+OMRProcessorDesc
+TR_RelocationRuntime::getProcessorDescription(TR_FrontEnd *fe, J9VMThread *curThread)
+   {
+   TR_ASSERT_FATAL(0, "Error: getProcessorDescription not supported in this relocation runtime");
+   return OMRProcessorDesc();
+   }
+
```

Comment:
maybe just move this to the header like you did for `TR_JITServerRelocationRuntime::getProcessorDescription`